lesson_06/homeworks06_LMS.py

```diff
@@ -0,0 +1,39 @@
+# ДЗ 6.1. Рахування унікальних символів в строці
+# Порахувати кількість унікальних символів в строці. Якщо їх більше 10 - вивести в консоль True, інакше - False.
+# Строку отримати за допомогою функції input()
+user_str = input('Enter a symbols:')
+
+if len(set(user_str)) > 10:
+    print("ДЗ 6.1. The number of unique characters is greater than 10: True")
```

Comment:
```suggestion
len_max = 10
otput = "ДЗ 6.1. The number of unique characters is greater than"
if len(set(user_str)) > len_max:
    print(f"{otput} {len_max}: True")
```
завжди думайте як зробити код більш універсальним